cicecore/cicedynB/general/ice_flux.F90

```diff
@@ -102,17 +124,32 @@ module ice_flux
        ! ice stress tensor in each corner of T cell (kg/s^2)
          stressp_1, stressp_2, stressp_3, stressp_4 , & ! sigma11+sigma22
          stressm_1, stressm_2, stressm_3, stressm_4 , & ! sigma11-sigma22
-         stress12_1,stress12_2,stress12_3,stress12_4    ! sigma12
+         stress12_1,stress12_2,stress12_3,stress12_4, & ! sigma12
+       ! ice stress tensor at U and T locations (grid_ice = 'CD') (kg/s^2)
```

Comment:
this comment is also for grid_ice = 'C', right?

Reply:
Fixed in upcoming PR.  I will look for other comments that say "CD" and change the to "C|CD".  I suspect these arose because we were only working on CD at first.